src/plugins/csv/csvplugin.cpp

```diff
@@ -43,62 +45,64 @@ CsvPlugin::CsvPlugin()
 
 bool CsvPlugin::write(const Map *map, const QString &fileName)
 {
+    // Extract file name without extension and path
+    QFileInfo fileInfo(fileName);
+    fileInfo.setCaching(false);
+    const QString fileNameWoExtension = fileInfo.baseName();
+    const QString filePath = fileInfo.path();
```

Comment:
To avoid confusion with actual file paths, I think this should just be called `path`.